Source/Scene/Expression.js

```diff
@@ -43,6 +43,23 @@ define([
         }
     };
 
+    var unaryFunctions = {
+        abs : Math.abs,
+        sqrt : Math.sqrt,
+        cos : Math.cos,
+        sin : Math.sin,
+        tan : Math.tan,
+        acos : Math.acos,
+        asin : Math.asin,
+        atan : Math.atan,
+        radians : function(left) {
```

Comment:
Here and below use the Cesium math functions: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/Math.js#L419-L440